src/dali/dali_main.py

```diff
@@ -162,6 +162,9 @@ def _dali_main_internal(country: AbstractCountry) -> None:
         with ThreadPool(args.thread_count) as pool:
             result_list = pool.map(_input_plugin_helper, input_plugin_args_list)
 
+        # prune None lists from the result_list
+        result_list = [[] if plugin_results is None else plugin_results for plugin_results in result_list]
```

Comment:
Is this necessary? The following line (extracting transactions) should work even with empty lists.

Reply:
There was a `NoneType` that kept popping up that I couldn't track down that was crashing the list comprehension in the next line, but I must have fixed it with the latest bug squashes. I took this line out.